doc/configuration.rst

```diff
@@ -1098,6 +1099,41 @@ optimize less but speed up the build time you could do::
 
 See ``$ optipng --help`` for a complete list of options.
 
+.. _image_srcset:
+
+Multi-resolution images
+=======================
+
+Web browsers allow a ``srcset`` parameter to the ``<img>`` tag that 
+allows the browser to support `responsive resolution images 
+<https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images>`__
+for hi-dpi/retina displays. Sphinx Gallery supports this via the ``image_srcset`` parameter::
+
+    sphinx_gallery_conf = {
+        ...
+        'image_srcset': ["", "2x"],
+    }
+
+that saves a 1x image at the normal figure dpi (usually 100 dpi) and a 2x 
+version at twice the density (e.g. 200 dpi).  The default is no extra images 
+(``'image_srcset': [""]``), and you can specify other resolutions if desired (e.g.
+``"1.5x"``).
+
+The matplotlib scraper creates a custom image directive in the rst file::
+
+    .. image-srcset:: /examples/images/sphx_glr_test_001.png
+        :alt: test
+        :srcset: /examples/images/sphx_glr_test_001.png, /examples/images/sphx_glr_test_001_2_0x.png 2.0x
+        :class: sphx-glr-single-img
+
+Which is converted to html by the custom directive as::
+
+    .. <img src="../_images/sphx_glr_test_001.png" alt="test", class="sphx-glr-single-img", 
+        srcset="../_images/sphx_glr_test_001.png, ../_images/sphx_glr_test_001_2_0x.png 2.0x>
+
+Note of course that this leads to a larger website, but responsive images will only serve the 
+appropriate-sized images to website visitors.
```

Comment:
Should maybe mention the things Eric said in https://github.com/sphinx-gallery/sphinx-gallery/pull/808#issuecomment-818043417

> We just have to clearly state in the docs that the sphinx_gallery.img_srcset extension only supports specific tags, and only HTML and LaTeX builders.

Reply:
Added:
```
Note that the ``.. image-srcset`` directive currently ignores other ``.. image`` directive 
tags like ``width``, ``height``, and ``align``.  It also only works with the *html* and 
*latex*  builders.  
```